src/frontend/css/style.css

```
@@ -0,0 +1,1264 @@
+  background: var(--color-bg-2);
+}
+
+.file-upload:hover {
+  border-color: var(--color-primary);
+  background: var(--color-bg-3);
+}
+
+.file-upload.dragover {
+  border-color: var(--color-primary);
+  background: var(--color-primary);
+  background: rgba(var(--color-teal-500-rgb), 0.1);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)